client/tsconfig.json

```
@@ -29,7 +29,8 @@
     "incremental": true
   },
   "exclude": [
-    "node_modules"
+    "node_modules",
+    "./src/styles/themes.ts"    
```

```suggestion
    "node_modules"
```
I don't think this should be ignored.